app/src/main/java/com/epam/themes/collectionviews/StudentsActivity.java

```diff
@@ -5,25 +5,51 @@
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.helper.ItemTouchHelper;
+import android.view.View;
 
 import com.epam.cleancodetest.R;
 import com.epam.themes.backend.IWebService;
 import com.epam.themes.backend.StudentsWebService;
 import com.epam.themes.backend.entities.Student;
+import com.epam.themes.collectionviews.recyclerview.ItemTouchCallbackStudent;
 import com.epam.themes.collectionviews.recyclerview.StudentsAdapter;
 import com.epam.themes.util.ICallback;
 
+import java.util.Arrays;
 import java.util.List;
+import java.util.Random;
 
 public class StudentsActivity extends AppCompatActivity {
 
     public static final int PAGE_SIZE = 10;
     public static final int MAX_VISIBLE_ITEMS = 40;
+    private static final List<String> STUDENTS_LIST = Arrays.asList(
```

Comment:
could be better to move all logic for storing editing and so on to webService. On UI layer just draw list. Think about splitting logic to layers.